spaces/S000052/README.md

```diff
@@ -9,8 +9,13 @@ refs:
     name: Counterexamples in Topology
   - wikipedia: Arithmetic_progression_topologies
     name: Arithmetic progression topologies on Wikipedia
+  - mathse: 4460847
+    name: The Kirch topology is the same as the prime integer topology
 ---
 Let $X=\mathbb{Z}^+$, the set of positive integers.  For $a,b \in X$, let $U_a(b) = \{b+na \in X : n \in \mathbb{Z}\} = (b+a\mathbb{Z})\cap X$.  Then $\{U_a(b) : a,b \in X, gcd(a,b)=1\}$ forms a basis for the relatively prime integer topology on $X$.
 
+Alternatively, a basis is given by $\{U'_a(b) : a,b \in X, gcd(a,b)=1\}$ with $U'_a(b) = \{b+na \in X : n \in \mathbb{N}\}$.
```

Comment:
There is a slight ambiguity here.  Some people define $\mathbb N$ to be $\\{0,1,\dots\\}$; others use $\\{1,2,\dots\\}$.
Writing `\{b+na  : n=0,1,2,\dots\}` would be an option. (note that we don't need `\in X` here, since the arithmetic sequence only goes to the right in this case).